core/src/main/scala/org/apache/spark/scheduler/TaskInfo.scala

```diff
@@ -30,6 +30,7 @@ import org.apache.spark.annotation.DeveloperApi
 @DeveloperApi
 class TaskInfo(
     val taskId: Long,
+    /** the index of this task in its TaskSet.  *Not* necessarily the same as the partitionid */
```

Comment:
Confused by your comments on this -- your comment in the thread for this PR seemed to imply that these are indeed different?